pkgs/data/fonts/cozette/default.nix

```
@@ -0,0 +1,60 @@
+{ stdenv, fetchurl, mkfontscale }:
+
+let
+  version = "1.5.1";
+  releaseUrl =
+    "https://github.com/slavfox/Cozette/releases/download/v.${version}";
+in stdenv.mkDerivation rec {
+  pname = "Cozette";
+  inherit version;
+
+  srcs = map fetchurl [
+    {
+      url = "${releaseUrl}/cozette.otb";
+      sha256 = "05k45n7jar11gnng2awpmc7zk9jdlzd6wz87xx49cp75jm4z9xm8";
+    }
+    {
+      url = "${releaseUrl}/CozetteVector.otf";
+      sha256 = "1sqhnjpizn1wi26lc7z2zml7yr7zkcpa72mh1drvd74rlcs1ip30";
+    }
+    {
+      url = "${releaseUrl}/CozetteVector.ttf";
+      sha256 = "1q4ml8shv9lmyc6bwhffwvbvl92s73j7xkb0rkqvci4f0zbz7mcy";
+    }
+  ];
+
+  nativeBuildInputs = [ mkfontscale ];
+
+  sourceRoot = "./";
+
+  unpackCmd = ''
+    otName=$(basename $(stripHash $curSrc))
```

The `basename` is redundant, also the variable must be quoted.
```suggestion
    otName=$(stripHash "$curSrc")
```